apps/server/src/modules/files-storage/entity/filerecord.entity.spec.ts

```diff
@@ -113,4 +118,21 @@ describe('FileRecord Entity', () => {
 			expect(securityCheck.requestToken).toEqual(securityCheck.requestToken);
 		});
 	});
+
+	describe('setName is called', () => {
+		const setup = () => {
+			const fileRecord = fileRecordFactory.build();
+			const newName = 'newName';
+
+			return { fileRecord, newName };
+		};
+
+		it('should set name', () => {
+			const { fileRecord, newName } = setup();
```

Comment:
I'm not in favor of this approach with the setup function. I think it adds unnecessary complexity and is in danger of mixing up with the before hooks. In our testing guideline there is also stated: "its okay to duplicate code for each test": https://hpi-schul-cloud.github.io/schulcloud-server/additional-documentation/nestjs-application/testing.html

Reply:
Please bring it up a general discussion about tests, because it's nothing we will change here.